ax-vasquez/drizzle-studio-expo
# update to expo 52; bump plugin version to 0.1.0

This is in response to the following issue:

* https://github.com/drizzle-team/drizzle-studio-expo/issues/4

Basically, after Expo updated to version 52, the typing for `SQLiteDatabase` used in each were no longer compatible. This PR simply updates Expo to 52, as well as updates requisite dependencies.

## Versioning Note

This plugin uses semantic versioning, but since only PATCH versions have been released, it wasn't clear to me how I should set the version. I've opted to bump the MINOR version since it's a breaking change, but I also didn't think it was enough to bump the MAJOR version.

## Testing

To test this:
1. From the root of this repo (and after installing all dependencies), run `npm run pack`
2. Open an existing Expo 52 application using SQLite in your IDE of choice
3. Copy the resulting `*.tgz` file into your project root
4. In your `package.json` file, replace the version string for the `expo-drizzle-studio-plugin` package with `file:<file_name>.tgz`
5. **You should not have to make any code changes beyond the package updates**
6. Run the application in either a simulator or your device
7. You should now be able to connect to the Drizzle studio
